InternTrack.Core.Api.Tests.Acceptance/InternTrack.Core.Api.Tests.Acceptance.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
both needs to be disabled this one and the one below it.

Reply:
changed to disable. But build fails. 
![Screenshot (48)](https://user-images.githubusercontent.com/120409509/235412464-0033e5cd-032d-4457-b035-baa4564b6799.png)